### PR TITLE
oc cp -> oc rsync

### PR DESCRIPTION
**Pull request by jpopelka**

### CODE REVIEW

File: sandcastle/api.py [hunk 1]
```diff
@@ -566,50 +566,31 @@ def _copy_path_to_pod(self, local_path: Path, pod_dir: Path):
...
+
+        run_command(
+            [
+                "oc",
+                "rsync",
+                exclude,
+                include,
+                "--progress=true",
+                f"--namespace={self.k8s_namespace_name}",
+                f"{local_path}/",  # ??? rsync doesn't work without the trailing /
```
Comment (TomasTomecek) [inline sandcastle/api.py:590]: yes, rsync is weird

File: sandcastle/api.py [hunk 2]
```diff
@@ -566,50 +566,31 @@ def _copy_path_to_pod(self, local_path: Path, pod_dir: Path):
...
-                    "-xzf",
-                    str(remote_tar_path),
-                    "-C",
-                    str(pod_dir),
-                ]
-                self._do_exec(unpack_cmd)
-            finally:
-                self._do_exec(["rm", "-rf", str(remote_tmp_dir)])
+        if local_path.is_dir():
+            exclude = "--exclude=lost+found"  # not needed
```
Comment (TomasTomecek) [inline sandcastle/api.py:573]: we can't even touch that

**Review by TomasTomecek — COMMENTED** [review]
nice :O

**Comment by TomasTomecek** [comment]
I'll try to review & run the tests locally tomorrow

**Comment by jpopelka** [comment]
Thanks, `test_ironman.py::test_from_pod` is failing for me locally, but it's failing in `master` as well.
I'll work on #64 meanwhile.

**Review by TomasTomecek — APPROVED** [review]
I found some free time b/w the breaks

LGTM, all tests passed for me (except for from_pod), let's give this a shot!

**Pull request merged**